From 34d22ab6cfd2dfe6dd171c8d0b31cafcdeb86298 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 29 Mar 2013 09:23:08 +0400
Subject: [PATCH 21/46] MIPS64: Default to N64 ABI

MIPS64 defaults to n32 ABI, this patch makes it
so that it defaults to N64 ABI

Signed-off-by: Khem Raj <raj.khem@gmail.com>

Upstream-Status: Inappropriate [OE config specific]
---
 gcc/config.gcc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index c835734..dd0739d 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -2017,29 +2017,29 @@ mips*-*-linux*)				# Linux MIPS, either endian.
 			default_mips_arch=mips32
 			;;
 		mips64el-st-linux-gnu)
-			default_mips_abi=n32
+			default_mips_abi=64
 			tm_file="${tm_file} mips/st.h"
 			tmake_file="${tmake_file} mips/t-st"
 			enable_mips_multilibs="yes"
 			;;
 		mips64octeon*-*-linux*)
-			default_mips_abi=n32
+			default_mips_abi=64
 			tm_defines="${tm_defines} MIPS_CPU_STRING_DEFAULT=\\\"octeon\\\""
 			target_cpu_default=MASK_SOFT_FLOAT_ABI
 			enable_mips_multilibs="yes"
 			;;
 		mipsisa64r6*-*-linux*)
-			default_mips_abi=n32
+			default_mips_abi=64
 			default_mips_arch=mips64r6
 			enable_mips_multilibs="yes"
 			;;
 		mipsisa64r2*-*-linux*)
-			default_mips_abi=n32
+			default_mips_abi=64
 			default_mips_arch=mips64r2
 			enable_mips_multilibs="yes"
 			;;
 		mips64*-*-linux* | mipsisa64*-*-linux*)
-			default_mips_abi=n32
+			default_mips_abi=64
 			enable_mips_multilibs="yes"
 			;;
 	esac
-- 
2.6.3

